### PR TITLE
Don't prebuffer when using PulseAudio.

### PR DESCRIPTION
**Pull request by padenot**
This makes the PulseAudio backend behave like all other backends.

@achronop r?

### CODE REVIEW

File: src/cubeb_pulse.c [hunk 1]
```diff
@@ -772,6 +787,8 @@ pulse_stream_init(cubeb * context,
     r = stream_update_timing_info(stm);
   }
 
+  WRAP(pa_stream_set_write_callback)(stm->output_stream, stream_write_callback, stm);
```
Comment (achronop) [inline src/cubeb_pulse.c]: You need to check again if output is requested `if (output_stream_params)`

Comment (achronop) [inline src/cubeb_pulse.c]: So write callback will not be called again after that point (stream in ready state) until stream is uncorked.

**Comment by achronop** [comment]
r+ with the output check.

**Comment by kinetiknz** [comment]
Have you testing this with A/V sync? It looks like get_position needs to be adjusted however many bytes we wrote in stream_silence_callback.

**Comment by padenot** [comment]
Yeah that's in the third patch.

**Comment by padenot** [comment]
This is a bad approach and does not really work, I'll do something better.

**Pull request closed without merging**